lardataalg/DetectorInfo/larproperties.fcl

```diff
@@ -18,8 +18,8 @@ standard_properties:
 
  # Optical properties
  # Fast and slow scintillation emission spectra, from [J Chem Phys vol 91 (1989) 1469]
```

Comment:
Is this comment line still correct with the updated params?

Reply:
Yes, the reference is correct, the problem is in the transcription of the measurements.

Reply:
approve